### ISSUE TITLE
Help with archiving OpenAire compliance

### ISSUE THREAD

**Issue by patrickda** [LibreCat/LibreCat#789]
Hi as you know we are up and running but still have some problems with our OpenAire Compliance:

1. Error with field Access Level:
It looks like LibreCat has it right (example https://research-explorer.app.ist.ac.at/oai?verb=GetRecord&metadataPrefix=oai_dc&identifier=oai:pub.research-explorer.app.ist.ac.at:1149) with the test we still got an error with 4927 from 5048 entries. 

2. Similar problem with field publication type with 4900 from 5048 entries. 

3. The field project identifier if filled with some values from project. Do we have to enter the funder name and project id in a very specify way? if yes it would be good to store this information separately because we get the input for this fields direct from our grant office

Thanks for any advice and help.

**Comment by vpeil** [comment]
@patrickda can you be more explicit, please? Your example from above looks correct to me.

**Comment by vpeil** [comment]
I'll close this one. If this is still a problem for you, please reopen and provide more details.

**Comment by patrickda** [comment]
Hi in between we found the problems:
 as documentation what we experienced (a few proposals for some small adoptions are inside) :
1. We introduced as new field OA for the OA status this had an impact on our export of the "Access Level".
2. We have some files with an embargo and the embargo end date was not exported to our OAI set. Maybe this can be added to the standard installation.
3. How we entered the project identifier didn't matched with the Open Aire recommendations. Maybe there could be a help text in LibreCat how the identifier have to be entered to fulfill the OpenAire compliance. 
4. We introduced some new publication types - his have to be matched to the correct OpenAire types.
